packages/core/test/display.test.ts

```diff
@@ -18,11 +18,11 @@ describe('iife/display.ts', () => {
         format: 'billboard',
         url: 'https://getpocket.com/',
         callbacks: {
-          click: 'http://example.com/click',
-          impression: 'http://example.com/impression',
-          report: 'http://example.com/report',
+          click: 'https://example.com/click',
```

Comment:
No substantial changes in this file, just doing `http` -> `https` find-replace for consistency/sanity.

Reply:
No longer true, we now have some new tests for the reporting flow down at the bottom